controllers/accounts.go

```diff
@@ -69,7 +71,7 @@ type RegistrationRecord struct {
 	Envelope *string `json:"envelope" validate:"required_without=SerializedRecord"`
 	// Serialized registration record
 	SerializedRecord *string `json:"serializedRecord" validate:"required_without_all=PublicKey MaskingKey Envelope"`
-	// Locale for verification email (optional, falls back to Accept-Language header)
+	// Locale for verification email
 	Locale string `json:"locale" validate:"max=8" example:"en-US"`
```

Comment:
To match my suggestion further down:
```suggestion
	Locale string `json:"locale" validate:"max=20" example:"en-US"`
```